exo-deploy/terraform/aws/network/bastion/vars.tf

```diff
@@ -0,0 +1,48 @@
+/* Variables */
+
+variable "availability_zones" {
+  type        = "list"
+  description = "List of availability zones to place subnets"
+}
+
+variable "env" {
+  description = "Environment tag, e.g prod"
+}
+
+variable "instance_type" {
+  description = "Instance type of the bastion hosts"
+  default     = "t2.micro"
+}
+
+variable "key_name" {
+  description = "Name of the key pair stored in AWS used to SSH into bastion instances"
+}
+
+variable "name" {
+  description = "Name tag, e.g stack"
+}
+
+variable "region" {
+  description = "Region of the environment, for example, us-west-2"
+}
+
+variable "subnet_ids" {
+  description = "ID's of the subnets to place instances into"
+  type        = "list"
+}
+
+variable "vpc_id" {
+  description = "ID of the VPC"
+}
+
+/* Output */
+
+output "ips" {
```

Comment:
don't like the idea of an easily guessable hostname for bastions, especially since we would want to reuse this setup for multiple partners/projects. outputting ssh config sounds good though